### DISCUSSION TITLE
can't see typed text on search box with javascript disabled

### DISCUSSION THREAD

**Discussion by ne555** [community/community#49691] [Discussions]
### Select Topic Area

Bug

### Body

with javascript disabled can't see whatever is typed in discussions-search-combobox
noticed that you had put .QueryBuilder-Input color into transparent
tried with light and dark deuteranopia themes
tested on firefox

no problem with issues tab search (js-issues-search)

**Comment by antn** [comment]
Sorry about that! This was a bug, but we have rolled out a fix. Please let us know if you're still having any trouble.

**Reply by ne555** [reply]
the problem persist
![bug](https://user-images.githubusercontent.com/1026012/225283757-c3546fdd-6073-461b-b12c-fa07dcf9dc63.png)
before «hello» it says «is: open»

**Reply by antn** [reply]
Hmm, that is odd! What browser are you using, and what version? 

Also, if you have any browser extensions, can you disable them temporarily to see if something might be interfering with the site? 

If you could also try it in another browser, if you have one installed, that would help narrow down if this is a problem with a specific browser or not.

**Reply by antn** [reply]
Sorry about that, I was able to reproduce this with JS disabled. Sorry for missing that - we had an unrelated issue recently that caused this even with JS enabled.

I'm going to check with our team to see if we can degrade this more gracefully. Although if you browse GitHub with JS disabled, a lot of things will not work properly currently.

**Reply by antn** [reply]
I've deployed a fix for this, so the text should be displayed when JS is disabled. Sorry for the trouble!
